tests/conftest.py

```diff
@@ -1489,7 +1489,9 @@ def cluster_teardown_finalizer():
     else:
         if config.ENV_DATA["platform"] == constants.IBMCLOUD_PLATFORM:
             ibmcloud.login()
-    if not config.ENV_DATA["skip_ocs_deployment"] and config.RUN["cephcluster"]:
+    if "cephcluster" not in config.RUN.keys():
```

Comment:
I think that it might be a problematic as we discussed yesterday and we don't need to do this for teardown at all.

@vavuthu tried to fix something like this here:
https://github.com/red-hat-storage/ocs-ci/pull/6187/files

So we don't need to most likely have rp_ocs_build attribute added for teardown at all, and do not cause the issues for teardown for problematic clusters.

Reply:
Issue is described here actually @shyRozen https://github.com/red-hat-storage/ocs-ci/issues/6186

Reply:
Hi @petr-balogh . At the of the error mentioned in #6186 check_clusters was trying to initiate LVM class and CephCluster class. I moved from this implementation in #6163 (because LVM class initiation was growing and now I only initiate OCP objects with kind "cephcluster" and "lvmcluster" and it has exception catching for every known error. The issue is not relevant anymore to the code that is in ocs-ci. This PR fixes some issue in check_clusters which should make it bulletproof I hope.

Reply:
ok, thanks for explanation. So let's see if we are good and we will keep in mind this change if not :)